modules/nf-core/gatk4/haplotypecaller/main.nf

```diff
@@ -16,9 +16,10 @@ process GATK4_HAPLOTYPECALLER {
     path  dbsnp_tbi
 
     output:
-    tuple val(meta), path("*.vcf.gz"), emit: vcf
-    tuple val(meta), path("*.tbi")   , optional:true, emit: tbi
-    path "versions.yml"              , emit: versions
+    tuple val(meta), path("*.vcf.gz")       , emit: vcf
+    tuple val(meta), path("*.tbi")          , optional:true, emit: tbi
+    tuple val(meta), path("*.realigned.bam"), optional:true, emit: bam
```

Comment:
sorry I am just thinking about this now, but shouldn't this be crams?

Reply:
option in gatk is to output bam for now, maybe in the future it'll be cram.
Anyway I think this is a very edgy situation, I see no issue into having bam here.
From what I read from the docs:
> ` Really for debugging purposes only. Note that the output here does not include uninformative reads so that not every input read is emitted to the bam. Turning on this mode may result in serious performance cost for the caller. It's really only appropriate to use in specific areas where you want to better understand why the caller is making specific calls.`

No one should really be doing that.
Let's keep it that way for now, and we'll improve later on

Reply:
ok makes sense 👍